pkgs/applications/misc/xdragon/default.nix

```
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, pkgconfig, gtk3 }:
+
+stdenv.mkDerivation rec {
+  pname = "xdragon";
+  version = "1.1.1";
+
+  src = fetchFromGitHub {
+    owner = "mwh";
+    repo = "dragon";
+    rev = "v${version}";
+    sha256 = "sha256-Kq/wiqj9I6W6BnX24Ar7Y4v7PAaXaXEwl80dANL4/zk=";
+  };
+
+  buildInputs = [ pkgconfig gtk3 ];
```

for cross compilation
```suggestion
  naviteBuildInputs = [ pkg-config ];
  buildInputs = [ gtk3 ];
```